README.md

```diff
@@ -1,17 +1,13 @@
-# LaunchDarkly Sample Flutter Application
+# LaunchDarkly sample Flutter application
 
 We've built a simple Flutter application that demonstrates how LaunchDarkly's SDK works.
 
-Below, you'll find the basic build procedure, but for more comprehensive instructions, you can visit the [Flutter SDK reference guide](https://docs.launchdarkly.com/sdk/client-side/flutter).
+Below, you'll find the basic build procedure. For more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Flutter SDK reference guide](https://docs.launchdarkly.com/sdk/client-side/flutter).
```

Comment:
```suggestion
Below, you'll find the build procedure. For more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Flutter SDK reference guide](https://docs.launchdarkly.com/sdk/client-side/flutter).
```
Your review of other variations of this work recommended to not keep both "simple" and "basic," so I've been removing the "basic".